tests/integration/test_numeric_array.py

```diff
@@ -0,0 +1,40 @@
+# encoding: utf-8
+
+"""Integration-test suite for numeric-arrays."""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import numpy as np
+
+from cr.cube.cube import Cube
+
+from ..fixtures import NA
+
+
+# pylint: disable=no-self-use
+class TestNumericArrays:
+    """Test-suite for numeric-arrays behaviors."""
+
+    def test_num_arr_x_cat_with_missings(self):
+        """Test means on numeric array, grouped by single categorical dimension."""
+        slice_ = Cube(NA.NUM_ARR_X_CAT_WITH_MISSINGS).partitions[0]
+
+        np.testing.assert_almost_equal(
+            slice_.means, [[87.66666667, 93.33333333, 1.0], [52.5, 50.0, 45.0]]
+        )
+        np.testing.assert_almost_equal(slice_.column_base, [5, 4, 2])
```

Comment:
Just by the first look, maybe we could use a different summary here, just to be explicit? Something like `mean_column_base`. Or even `mean_unweighted_column_base`. This is potentially important, because we're going to have different measures in the cube (maybe multiple), and will potentially also need different bases for them.

Reply:
not sure because the `column_base` is a measure that we have for all the kind of cube.